### PR TITLE
Setup tests

### PR DESCRIPTION
**Pull request by maurodoglio**
This adds a basic setup to run tests using [pytest](http://pytest.org).
It also adds histogram_tools.py to this repository, so from now on we need to bump up the minor version of this library everytime that file is updated on mozilla-central.

### CODE REVIEW

**Comment by vitillo** [comment]
OK, let me know when this is ready to be reviewed/merged.

**Comment by maurodoglio** [comment]
We have a green run!
Caching the pip packages will make the travis run go from [7'45"](https://travis-ci.org/mozilla/python_moztelemetry/builds/125574800) to  about [1'](https://travis-ci.org/maurodoglio/python_moztelemetry) !

**Comment by vitillo** [comment]
Nice! It seems that currently `histogram_tools.py` is not fetched when the package is built. Could you add a hook for it?

**Comment by maurodoglio** [comment]
@vitillo I added histogram_tools.py to the repo in the first commit. It should be there when you install the package.

**Comment by vitillo** [comment]
Right, I was just wondering if we should setup a hook that fetches the current `histogram_tools.py` from mozilla-central when the package is being tested locally. Maybe we can address this in another PR.

**Comment by maurodoglio** [comment]
The second commit adds a script to update histogram_tools.py from mozilla-central.

File: README.md [hunk 1]
```diff
@@ -14,3 +14,10 @@ password:example_pass
 ```
 - Fetch the latest code with `git pull`
 - Update PyPI with `python setup.py sdist upload`
+
+
+## Updating histogram_tools.py
+moztelemetry/histogram_tools.py is a mirror of its counterpart from
+[mozilla-central](https://hg.mozilla.org/mozilla-central/raw-file/tip/toolkit/components/telemetry/histogram_tools.py).
+To update it to the latest version you can run `update_histogram_tools`.
+Such command is available only if you install the moztelemetry package using `python setup.py install`.
```
Comment (vitillo) [inline README.md]: Does this work with pip's [editable install](https://pip.pypa.io/en/stable/reference/pip_install/#editable-installs) as well?

Comment (maurodoglio) [inline README.md]: Yeah it works with editable installs as well. I'll update the readme.

Comment (maurodoglio) [inline README.md]: As an alternative, I can wrap the update_histogram_tools in a python standalone script and put it in a bin directory at the root of the repository. Do you have a preference?

Comment (vitillo) [inline README.md]: Yeah, your that makes more sense to me as the script is something we want to use only during development.

**Comment by vitillo** [comment]
Looks good, thanks!

**Pull request merged**